metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://angularlogos.com/assets/metadata-schema.json",
```

Comment:
Is there a way to defined in this schema that certain properties are optional?

Reply:
Yep, all properties are optional (it works the other way around, you define the required props)